src/handlers/payout/index.ts

```diff
@@ -0,0 +1,44 @@
+import { getWalletAddress } from "../../adapters/supabase";
+import { getBotContext } from "../../bindings";
+import { addCommentToIssue, generatePermit2Signature } from "../../helpers";
+import { Payload } from "../../types";
+import { bountyInfo } from "../wildcard";
+
+export const handleIssueClosed = async () => {
+  const context = getBotContext();
+  const { log } = context;
+  const payload = context.payload as Payload;
+  const issue = payload.issue;
+  if (!issue) return;
+
+  log.info(`Handling issues.closed event, issue: ${issue.number}`);
+  const issueDetailed = bountyInfo(issue);
+  if (!issueDetailed.isBounty) {
+    log.info(`Skipping... its not a bounty`);
+    return;
+  }
+
+  const assignees = issue?.assignees ?? [];
+  const assignee = assignees.length > 0 ? assignees[0] : undefined;
+  if (!assignee) {
+    log.info("Skipping to proceed the payment because `assignee` is undefined");
+    return;
+  }
+
+  if (!issueDetailed.priceLabel) {
+    log.info("Skipping to proceed the payment because price not set");
+    return;
+  }
+
+  const priceInETH = issueDetailed.priceLabel!.substring(7, issueDetailed.priceLabel!.length - 4);
```

Comment:
```suggestion
  const priceInEth = issueDetailed.priceLabel!.substring(7, issueDetailed.priceLabel!.length - 4);
```

If we do the same code style as the dollar repo, its similar to us renaming all identifiers with `NFT` to `Nft`

Reply:
https://github.com/ubiquity/bounty-bot/pull/138/commits/0a2f5d727febbba6a726141ec04f0e1f83ecb6ab